src/main/webapp/search-script.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Initializes page. */
+function init() {
+  loadSearchListPage();
+}
+
+/**
+ * Loads the search results with names that match the searched text.
+ */
+function loadSearchListPage() {
+  const schoolsDataList = JSON.parse(localStorage.getItem('schoolsDataList'));
```

Comment:
can we show a message or a `li` saying that "Empty result" when there are nothing in the localStorage? This can be a `//TODO`.